openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -10,15 +11,18 @@
 from edx_oauth2_provider.tests.factories import ClientFactory
 from provider.constants import CONFIDENTIAL
 
+from certificates.models import CertificateStatuses, GeneratedCertificate  # pylint: disable=import-error
+from lms.djangoapps.certificates.tests.factories import GeneratedCertificateFactory
```

Comment:
To get around an error about conflicting models documented [here](https://code.djangoproject.com/ticket/22280). Importing the models from `lms.djangoapps.certificates.models` raises a `RuntimeError`. I don't like it either, but I'm having a hard time understanding how to fix the problem. It looks like this isn't a new issue, either: https://github.com/edx/edx-platform/search?l=python&q=%22certificates.models%22&utf8=%E2%9C%93.